Liadi/events-manager
# #152965693 fix template issues

#### What does this do
 - Fix UX issues (app navigation is more intuitive and organised)
 - Add footer to all pages
 - Add theme across pages (app color: orange)
 - Add simple dashboard
 - Fix "create event page" organisation (based on feedback from LFA)
 - Fix page styling issues

#### Task to be completed
 - Slide in center details page(slated-events.html) has a padding-left issue
 - For the footer, viewport height(vh) is used to style the page to keep the footer at the bottom of page regardless of page content height(this gives a slight glitch on very small screen)

#### How to test features
 - Navigate to https://liadi.github.io/events-manager/template/index.html
 - Test responsiveness using developer tools
 OR
 - Clone repo(for commit history)
 - Ensure branch is 'bg-template-issues' 
 - Open browser
 - Navigate to [repo base path]/template/index.html
 
#### Background context
 - Issues fixed are from feedback

#### Related PiivotalTracker stories
 - #152965693 (and all other template related stories)

#### Screenshots
-------------------------------------
###### Dashboard
![bg2](https://user-images.githubusercontent.com/29649308/33006001-da5b4366-cdc8-11e7-91dd-66cd51b53050.PNG)
-------------------------------------------
###### Create an event
![bg3](https://user-images.githubusercontent.com/29649308/33006068-3a382da8-cdc9-11e7-9f94-b6864f14c3be.PNG)
------------------------------------------------
###### Centers
![bg4](https://user-images.githubusercontent.com/29649308/33006153-84faba22-cdc9-11e7-9114-b43d3ce29b15.PNG)
-----------------------------------------------
###### Add center
![bg5](https://user-images.githubusercontent.com/29649308/33006203-bd79a052-cdc9-11e7-9bf3-0d9014c27f00.PNG)

-----------------------------------------------------
###### Manage events
![bg6](https://user-images.githubusercontent.com/29649308/33006244-ec0065e6-cdc9-11e7-9943-d4f558cd3a48.PNG)
-----------------------------------------------------------
###### Carousel padding issue
![bg1](https://user-images.githubusercontent.com/29649308/33005785-b70f293c-cdc7-11e7-8bda-529fa9e5f8d4.PNG)
-----------------------------------------------------------------------------------------

#### Questions
 - Any suggestion as regards keeping the footer at the bottom of page?
